helper-cli/src/main/kotlin/commands/ExportLicenseFindingCurationsCommand.kt

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (C) 2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.helper.commands
+
+import com.beust.jcommander.JCommander
+import com.beust.jcommander.Parameter
+import com.beust.jcommander.Parameters
+
+import com.here.ort.helper.CommandWithHelp
+import com.here.ort.helper.common.RepositoryLicenseFindingCurations
+import com.here.ort.helper.common.getRepositoryLicenseFindingCurations
+import com.here.ort.helper.common.mergeLicenseFindingCurations
+import com.here.ort.model.OrtResult
+import com.here.ort.model.readValue
+import com.here.ort.model.yamlMapper
+import com.here.ort.utils.PARAMETER_ORDER_MANDATORY
+import com.here.ort.utils.PARAMETER_ORDER_OPTIONAL
+import com.here.ort.utils.safeMkdirs
+
+import java.io.File
+
+@Parameters(
+    commandNames = ["export-license-finding-curations"],
+    commandDescription = "Export the license finding curations to a license finding curations file which maps "
```

Comment:
Nit: change " to a license finding curations file" to simpler "to a file"

Reply:
Done.